sensing/pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp

```diff
@@ -70,6 +76,22 @@ void DistortionCorrectorComponent::onVelocityReport(
   }
 }
 
+void DistortionCorrectorComponent::VehicleReportCheck(DiagnosticStatusWrapper & stat)
+{
+  stat.add("Topic publisher number ", std::to_string(velocity_report_sub_->get_publisher_count()));
+  auto level = DiagnosticStatus::OK;
+  std::string msg;
+  if (velocity_report_sub_->get_publisher_count() == 0) {
```

Comment:
[MUST] Two not good point. 
1. Checking publisher count is not good way since there might be publisher node which doesn't not really publish topics. The actual queue size of subscribed topic must be checked.
2. Checking for topics asynchronously with distortion correction is not a correct diagnosis.